odoo/addons/base/tests/test_user_has_group.py

```diff
@@ -33,13 +33,13 @@ def setUp(self):
         self.grp_public = self.env.ref(self.grp_public_xml_id)
 
     def test_env_uid(self):
-        Users = self.env['res.users'].with_user(self.test_user)
+        self_user = self.env['res.partner'].with_user(self.test_user)
```

Comment:
The name `self_user` is confusing, because it's an empty partner that has nothing to do with `self` :laughing: 
```suggestion
        partner = self.env['res.partner'].with_user(self.test_user)
```